deploy/charts/operator/README.md

```diff
@@ -92,4 +92,11 @@ The command removes all the Kubernetes components associated with the chart and
 | operator.toolhiveRunnerImage | string | `"ghcr.io/stacklok/toolhive/proxyrunner:v0.3.3"` | Image to use for Toolhive runners |
 | operator.volumeMounts | list | `[]` | Additional volume mounts on the operator container |
 | operator.volumes | list | `[]` | Additional volumes to mount on the operator pod |
+| registryAPI | object | `{"image":"ghcr.io/stacklok/toolhive/thv-registry-api:latest","serviceAccount":{"annotations":{},"automountServiceAccountToken":true,"labels":{},"name":"toolhive-registry-api"}}` | All values for the registry API deployment and associated resources NOTE: Registry API ServiceAccount and RBAC resources are only created when operator.features.experimental is enabled |
```

Comment:
should we mention here the experimental flag or is it a generated file?

Reply:
it's a generated file